libs/shared/problem/src/problems.ts

```diff
@@ -39,8 +39,8 @@ export interface AttemptFailedProblem extends BaseProblem {
 }
 
 export interface ProviderErrorReason {
-  title: StaticText
-  summary: StaticText
+  title: StaticText & { values?: Record<string, string> }
+  summary: StaticText & { values?: Record<string, string> }
```

Comment:
Is this required?

StaticText is already defined as:

```typescript
export type RecordObject<T = unknown> = Record<string, T>
export type StaticTextObject = MessageDescriptor & {
  values?: RecordObject<any>
}

export type StaticText = StaticTextObject | string
```

Reply:
You are right, it is unnecessary - removed the change